templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
   </table>
 
   {if !$contributionMode}
-    <fieldset class="payment-details_group">
-      <legend>
-        {ts}Payment Details{/ts}
-      </legend>
-        <table class="form-layout-compressed" >
```

Comment:
Anything in this table will not show on the NEW contribution form (I actually think it's kinda tidy the way it works)